v4/newrelic/application.go

```diff
@@ -104,5 +109,9 @@ func NewApplication(opts ...ConfigOption) (*Application, error) {
 	if nil == tracer {
 		tracer = global.Tracer("traceName")
 	}
-	return &Application{tracer: tracer}, nil
+	propagators := propagation.New(
+		propagation.WithInjectors(trace.TraceContext{}),
+		propagation.WithExtractors(trace.TraceContext{}))
```

Comment:
Just thinking aloud here. In the future it might be nice to allow customers to configure which injector/extractor they use. We could put the `trace.TraceContext` as a field on the `Config.OpenTelemetry` config option. Something like that.

I say this because a while ago I implemented a B3 propagation integration for the agent. In v4 of the agent, we could deprecate that integration and instead allow customers to specifically set their injector/extractor.

Reply:
Done. I put the propagators into the config, along with the tracer. This should give users full flexibility. I also added a B3 test.